sdk/handleError.js

```diff
@@ -15,14 +15,23 @@ class APIError extends Error {}
 function validateResponse(response) {
   const errorCode = get(response, 'body.errorCode');
   if (errorCode) {
-    throw new APIError(`Error: ${translateErrorCode(errorCode)}`);
+    const readableErrorMessage = maybeTranslateErrorCode(errorCode);
+    if (readableErrorMessage) {
+      throw new APIError(`Error: ${readableErrorMessage}`);
+    }
   }
 
-  const errorText = get(response, 'body.error');
+  // An error is anything with an error message, or any error code
+  // with an untranslated message.
+  const errorText = get(response, 'body.error') || (errorCode && get(response, 'body.message'));
   if (errorText) {
     throw new APIError(errorText);
   }
 
+  if (errorCode) {
+    throw new APIError(`Error: ${errorCode}`);
```

Comment:
In the worst case (untranslatable error code with no message) we can still `grep ERROR_CODE` in our code base when we try helping a user.